chef/site-cookbooks/wca/recipes/default.rb

```diff
@@ -323,7 +323,8 @@
 end
 
 #### Initialize rails gems/database
-execute "bundle install --without none" do
+execute "bundle install #{"--deployment" if rails_env == "production"} --path /home/#{username}/.bundle" do
```

Comment:
So I have no idea why but starting up a VM in "development" environment with "--without production" fails at the db:setup [here](https://github.com/cubing/worldcubeassociation.org/blob/a3948bde75de9cce3233192c8292d791399d871d/chef/site-cookbooks/wca/recipes/default.rb#L334) :(
I'm going to see how it behaves starting a production VM...